go.mod

```diff
@@ -1,28 +1,115 @@
-module github.com/heptiolabs/eventrouter
+module github.com/harvester/eventrouter
```

Comment:
Wondering if we could keep the original module name?

Reply:
I suppose the upstream keeps the original module name so that the code works fine after pulling upstream commits.
The suggestion here is to change it to Harvester's since upstream has archived it.

Reply:
what do u guys think @bk201 @futuretea

Reply:
What happens if we don't change it to harvester